docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,185 @@
 # Arcaflow Getting Started Guide
 
-## Step 1: Get a container engine
+## Running Workflows
+An Arcaflow workflow is a definition of steps structured together to perform complex actions. Workflows are defined as machine-readable YAML and therefore can be version-controlled and shared easily to run in different environments. A workflow is a way of encapsulating and sharing expertise and ensuring reproducible results.
 
-In order to use Arcaflow, you will need a container engine on your computer. For the purposes of this guide, we'll assume you are using Docker or Podman.
+Running a workflow only requires having the [Arcaflow engine binary](https://github.com/arcalot/arcaflow-engine/releases), the workflow definition file, and in most cases an input file. An optional config file also allows for setting workflow defaults, such as log levels. All of these files are typically defined in YAML. Additionally, the target of the workflow needs a compatible container platform, such as Podman, Docker, or Kubernetes.
 
-## Step 2: Get the engine
+!!! note
+    The default container platform for the Arcaflow engine is Podman. To use another platform, a [configuration file](/arcaflow/running/setup/#configuration) is required.
```

Comment:
I don't think we have an exact issue for it. There is a related issue: https://github.com/arcalot/arcaflow-engine/issues/191

This will likely require that we expand the deployer interface to add a way to detect if the deployer is available. In that case, we can update the config to support ranked deployers based on availability.